src/vs/workbench/api/node/extHost.api.impl.ts

```diff
@@ -329,16 +329,20 @@ export function createApiFactory(
 				return extHostTerminalService.onDidCloseTerminal(listener, thisArg, disposables);
 			},
 			showInformationMessage(message, first, ...rest) {
-				return extHostMessageService.showMessage(Severity.Info, message, first, rest);
```

Comment:
Since this change broke things, I'm going to revert this file and open an issue so that someone more knowledgeable can take a look at the proper fix